source/_integrations/shelly.markdown

```diff
@@ -69,36 +69,31 @@ Shelly generation 2 devices running firmware 0.12 or later can act as a Bluetoot
 
 ## Entity naming (generation 1)
 
-The integration uses `Device Name` to name its entities if the device has only one relay or no relays at all.
-
-The integration uses the following strategy to name its entities if the device has more than one relay:
-
-- If `Device Name` or `Channel Name` is set in the device, the integration will use them to generate the entities' name.
-- If channel names are set, they will be used in the entity names. The device name will not be used.
-- If only the device name is set, and the device has multiple channels, the channel number will be appended to the entity name (e.g., Channel 2).
-- In case device name and channel names are not set, the entity name will be generated by the `Device Type`, `Device ID` and `Channel Number`.
-
-Examples:
+The integration uses the following strategy to name its entities:
 
-| Device Name | Channel Name   | Entity Name                     |
-| ----------- | -------------- | ------------------------------- |
-| `Not set`   | `Not Set`      | shellyswitch25-ABC123 Channel 1 |
-| `Not set`   | Kids Room Bulb | Kids Room Bulb                  |
-| Kitchen     | `Not Set`      | Kitchen Channel 1               |
-| Bedroom     | Round Bulb     | Round Bulb                      |
+- If `Device Name` and `Channel Name` are set in the device, the integration will use them to generate the entities' name, e.g. `Shelly 2.5 Kitchen Light`
+- If `Device Name` or `Channel Name` are set to the default value, the integration will use the `Device ID` or default channel name to generate the entities' name, e.g. `Shelly2.5-9808D1D8B912 Channel 0`.
```

Comment:
IMHO, is not clear what happens when one of the 2 is not set to the default value.

Reply:
I made a small change to the content. Do you have a suggestion what we can add more here? I prefer a short phrases because users do not read long content anyway ;)

Reply:
The phrase wording **"will use the `Device ID` or default channel name"**, is not clear to me: which one ? when ? why one and not the other ?
Maybe I'm too picky, but this is how I get it.
Simone

Reply:
Sorry, I don't understand your questions. IMO, the sentence is simple and understandable. If you have a better idea how to describe it, I'm open to suggestion.